app/helpers.php

```diff
@@ -73,6 +73,17 @@ function format_date_form_field($model, $field, $defaut = 'MM/DD/YYYY')
 }
 
 /**
+ * Formate a timestamp using Carbon for display in a view.
+ *
+ * @param  string  $timestamp
+ * @return /Carbon/Carbon
+ */
+function format_timestamp_for_display($timestamp)
```

Comment:
You end up using this function on the `updated_at` field which is already a `Carbon` instance (at least, it should be) and then in this function you make a new Carbon instance out of it again. Is that on purpose?